dist/oboe-browser.js

```diff
@@ -2606,7 +2606,9 @@ function applyDefaults( passthrough, url, httpMethodName, body, headers, withCre
          // Default Content-Type to JSON unless given otherwise.
          headers['Content-Type'] = headers['Content-Type'] || 'application/json';
       }
-      headers['Content-Length'] = headers['Content-Length'] || body.length;
+       //Issue #146 (https://github.com/jimhigson/oboe.js/issues/146)
+      //see https://stackoverflow.com/a/7210840/3501729
+      //headers['Content-Length'] = headers['Content-Length'] || body.length;
```

Comment:
Could you revert these changes? I think it's best to not commit changes in `dist/` until a new version is being pushed, but I haven't improved the tooling to do that 😅